.gitlab-ci.hpc-spack.yml

```
@@ -0,0 +1,28 @@
+include:
+  - project: hpc/gitlab-pipelines
+    file: spack-build-components.gitlab-ci.yml
+
+default:
+  tags: []
+
+spack-setup:
+  tags: [bb5_map]
+spack-build:
+  tags: [bb5_map]
```

```suggestion
spack-build:
```
This should be the default again, too